tests/test_resource.py

```diff
@@ -330,6 +334,31 @@ def test_options_request(self):
             ALLOW_ORIGIN_HEADER: ORIGIN_EXAMPLE,
         }
 
+    def test_cors_headers_param_request(self) -> None:
```

Comment:
It would be also good to add a test that shows that headers from the parameter are more important than the ones from the env variable - as we saw, is easy to change this order.